packages/faceted-search/src/dictionary/operator.dictionary.js

```diff
@@ -1,6 +1,6 @@
 const operatorNames = {
 	contains: 'contains',
-	equal: '=',
+	equal: 'equal',
```

Comment:
I've implemented something like that [here](https://github.com/Talend/ui/blob/master/packages/forms/src/UIForm/fields/Comparator/Comparator.component.js#L13)
We definitively capitalize somewhere else this kin of mapping!

Reply:
Thx for show me this, we can't support all operators for now so it's good here to have only the one that we implement, but yes it's a maybe good idea. 
As faceted-search is another package that ui and can evolve in another way, maybe it's more simple to not share this object for now

Reply:
👌 just a comment to think about it!